src/components/Panels/PropertiesPanel/PropertiesPanel.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div
+        id="moduleProperty"
+        class="moduleProperty noSelect effect1 properties-panel draggable-panel draggable-panel-css guide_2"
+    >
+        <PanelHeader :header-title="$t('simulator.panel_header.properties')" />
+        <div class="panel-body">
+            <div id="moduleProperty-inner">
+                <div id="moduleProperty-header">{{ panelBodyHeader }}</div>
+                <PanelType1 v-if="panelType == 1" />
+                <PanelType2 v-if="panelType == 2" />
+                <PanelType3
+                    v-if="panelType == 3"
+                    :key="toRaw(propertiesPanelObj)"
+                    :data="toRaw(propertiesPanelObj)"
```

Comment:
why was toRaw necessary here?

Reply:
assigning an object to data property it converts into proxy object.
I found out Vue 3 is using [ES6 Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) to make an objects reactive
I was not able to compare `propertiesPanelObj` with `simulationArea.lastselected` to prevent updating panel type every time.
So used `toRaw()` to return the original object from proxy maintaining reactivity.